tests/unit_tests/test_mesh.py

```diff
@@ -1,6 +1,10 @@
-import openmc
-import pytest
+from math import pi
+
 import numpy as np
+import pytest
+
+import openmc
```

Comment:
From the perspective of tests, `openmc` is a third-party package, not the local package
```suggestion
import pytest
import openmc
```

Reply:
Thanks Paul, I trusted [isort](https://pycqa.github.io/isort/) a bit much here